### PR TITLE
Journey home 1

### PR DESCRIPTION
**Pull request by thiagogvasc**

### CODE REVIEW

File: src/pages/api/user/[id]/experience/index.js [hunk 1]
```diff
@@ -23,18 +24,33 @@ export default async function handler(req, res) {
       // Create an experience for an user and post it to the database
       case 'POST':
         const experience = new ExperienceModel({
```
Comment (iamdavidmichaelmoore) [inline src/pages/api/user/[id]/experience/index.js:26]: Is this going to include the two submodules that we talked about? See [Backend Meeting](https://docs.google.com/document/d/1ptGG2C2oB9NXGKJC0VPB-p_FbxdNwkj6VrEdm0lVwok/edit?usp=sharing) Notes from Nov. 4.

File: src/pages/api/user/[id]/experience/index.js [hunk 2]
```diff
@@ -23,18 +24,33 @@ export default async function handler(req, res) {
...
+          title: req.body.title,
           content: req.body.content,
+          category: req.body.category,
           skills: req.body.skills
         })
     
         try {
```
Comment (iamdavidmichaelmoore) [inline src/pages/api/user/[id]/experience/index.js:33]: This is a good practice!

File: src/pages/experience/index.js [hunk 3]
```diff
@@ -13,13 +13,74 @@ import Button from '@mui/material/Button'
 
 import JourneySideBar from 'components/JourneySideBar'
 
+import axios from 'axios'
+
 
 const drawerWidth = 240
 
 const ExperiencePage = () => {
+
```
Comment (iamdavidmichaelmoore) [inline src/pages/experience/index.js:25]: All of this looks good. Just make sure to comment on your code.

File: src/views/Home/Home.js [hunk 4]
```diff
@@ -67,7 +67,7 @@
```
Comment (iamdavidmichaelmoore) [inline src/views/Home/Home.js]: Same as the previous file. Comment the code.

File: src/views/Home/Home.js [hunk 5]
```diff
@@ -104,50 +104,85 @@ import Form from './components/Form'
...
-      ]
-    }
-  ]
+      })
+    })
+    return categories
+  }
 
-  const drawerWidth = 240;
+  const categories = getAllCategories()
```
Comment (iamdavidmichaelmoore) [inline src/views/Home/Home.js]: What's the intended implementation for `getAllCategories()`?

**Review by iamdavidmichaelmoore — APPROVED** [review]

**Comment by iamdavidmichaelmoore** [comment]
@thiagogvasc Make the necessary changes per the comments and merge it tonight with the main branch.

**Comment by iamdavidmichaelmoore** [comment]
@thiagogvasc Make the necessary changes per the comments and merge with the main branch tonight as soon as possible.

File: src/views/Home/Home.js (hunk 5 above)
Comment (thiagogvasc) [reply src/views/Home/Home.js]: I ran into some problems with the categorization of the experiences. I dropped the function and forgot to remove it. Disregard

File: src/pages/api/user/[id]/experience/index.js (hunk 1 above)
Comment (thiagogvasc) [reply src/pages/api/user/[id]/experience/index.js:26]: Not for this milestone. For the showcase module I would have to implement media storage management (images, videos, docs, etc.), which I didn't have time to do for this milestone.

Comment (iamdavidmichaelmoore) [reply src/pages/api/user/[id]/experience/index.js:26]: Understood. Thanks!

File: src/views/Home/Home.js (hunk 5 above)
Comment (iamdavidmichaelmoore) [reply src/views/Home/Home.js]: Thanks for clarifying that.

**Pull request merged**